src/automation/azext_automation/manual/_params.py

```diff
@@ -131,3 +132,86 @@ def load_arguments(self, _):
+
+    with self.argument_context('automation schedule update') as c:
+        c.argument('description', help='The description of the schedule.')
+        c.argument('is_enabled', arg_type=get_three_state_flag(), help='Indicating whether this schedule is enabled.')
+
+    with self.argument_context('automation schedule list') as c:
+        c.argument('automation_account_name', help='The name of the automation account.', id_part=None)
+
+    with self.argument_context('automation software-update-configuration') as c:
+        c.argument('automation_account_name', help='The name of the automation account.')
+        c.argument('software_update_configuration_name', options_list=['--name', '-n', '--configuration-name'],
+                   help='The name of the software update configuration.')
+
+    with self.argument_context('automation software-update-configuration create') as c:
+        c.argument('operating_system', arg_type=get_enum_type(['Windows', 'Linux']),
+                   help='Operating system of target machines.')
+
+        c.argument('included_update_classifications',
+                   arg_type=get_enum_type(["Unclassified", "Critical", "Security", "UpdateRollup", "FeaturePack",
+                                           "ServicePack", "Definition", "Tools", "Updates"]),
+                   help='Update classification included in the software update configuration.'
+                        ' A comma separated string with required values.')
+        c.argument('excluded_kb_numbers', nargs='+', help='KB numbers excluded from the software update configuration.')
+        c.argument('included_kb_numbers', nargs='+', help='KB numbers included from the software update configuration.')
+        c.argument('reboot_setting', help='Reboot setting for the software update configuration.')
+        c.argument('duration', validator=validator_duration,
+                   help='Maximum time allowed for the software update configuration run.'
+                        ' Duration needs to be specified using the format PT[n]H[n]M[n]S as per ISO8601.')
+        c.argument('azure_virtual_machines', nargs='+', help='List of azure resource Ids for azure virtual machines'
+                                                             ' targeted by the software update configuration.')
+        c.argument('non_azure_computer_names', nargs='+',
+                   help='List of names of non-azure machines targeted by the software update configuration.')
+        c.argument('azure_queries_scope', nargs='+',
+                   help='List of Azure queries scope in the software update configuration.')
+        c.argument('azure_queries_locations', nargs='+',
+                   help='List of Azure queries location in the software update configuration.')
+        c.argument('tags', nargs='+',
+                   help='List of Azure queries tag settings in the software update configuration.')
```

Comment:
Could you use `tags_type` as the `arg_type` ?

Reply:
Here `tags` and `tag_type` are different structures

Reply:
It is suggested to make the parameter another name to avoid making the user misunderstand its function